Description: fix interpreters in contrib python scripts
 Upstream is not interested in this patch and considers it Debian-specific.
Author: Andriy Senkovych <jolly_roger@itblog.org.ua>
Last-Update: 2011-03-02
--- a/contrib/bk_buildbot.py
+++ b/contrib/bk_buildbot.py
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python
+#!/usr/bin/python
 #
 # BitKeeper hook script.
 #
--- a/contrib/bzr_buildbot.py
+++ b/contrib/bzr_buildbot.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 # Copyright (C) 2008-2009 Canonical
 #
 # This program is free software: you can redistribute it and/or modify
--- a/contrib/googlecode_atom.py
+++ b/contrib/googlecode_atom.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 # GoogleCode Atom Feed Poller
 # Author: Srivats P. <pstavirs>
 # Based on Mozilla's HgPoller
--- a/contrib/run_maxq.py
+++ b/contrib/run_maxq.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env jython
+#!/usr/bin/env python
 
 import sys
 import glob
--- a/contrib/webhook_status.py
+++ b/contrib/webhook_status.py
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 import urllib
 
 from twisted.python import log
